docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -82,6 +90,8 @@ ransomware families — including those targeting the system’s master boot rec
 Ransomware protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license].
 If you upgrade to a Platinum or Enterprise license from Basic or Gold, ransomware protection will be disabled by default.
 
+When ransomware protection is enabled, canary files are placed in targeted locations throughout the Windows file system. They provide an early warning system for potential ransomware activity. When canary files are modified, Elastic Defend will immediately generate a ransomware alert. If *prevent* ransomware is active, {elastic-defend} will terminate the process responsible for modifying the file.
```

Comment:
Some suggestions to link the first and second sentences and address some of the passive voice.
  
```suggestion
When ransomware protection is enabled, canary files are placed in targeted locations throughout the Windows file system as an early warning system for potential ransomware activity. When canary files are modified, Elastic Defend immediately generates a ransomware alert. If *prevent* ransomware is active, {elastic-defend} terminates the process that modified the file.
```

Reply:
For me is not clear what `targeted locations` means. Is there a specific directory in mind where we locate the canary files? I might be good to document this directory for Windows? And we might point out a bit more clear that canary files are not available on MacOS and Linux.

Reply:
@roxana-gheorghe Can you provide any insight into Andre's question about the location of canary files?

Reply:
Do we want to publicize where we're placing the canary files? Would that make it easier for adversaries to evade the warning system we're putting in pace? I'm not trying to be contrary here. I'm just wondering if it's safer to obscure these details in the public docs.

Reply:
+1 to @nastasha-solomon's edits to reduce passive voice, but that first sentence is pretty long now. Is "throughout the Windows file system" necessary? or maybe shorten to "on the host"?

On publicizing the location: this doesn't seem to identify specific locations, just that the files are out there somewhere.